client/src/stores/notificationsStore.ts

```diff
@@ -12,8 +12,8 @@ import { mergeObjectListsById } from "@/utils/utils";
 
 import { useBroadcastsStore } from "./broadcastsStore";
 
-const ACTIVE_POLLING_INTERVAL = 5000;
-const INACTIVE_POLLING_INTERVAL = 30000;
+const ACTIVE_POLLING_INTERVAL = 60000; // 1 minute
```

Comment:
I think 30 seconds might also be fine, that's still 6 times less frequently ? At which point do you think admins would get confused with notifications not showing ?

Reply:
Ok, we can set it to 30 and see how it goes